src/iuse_actor.cpp

```diff
@@ -2183,13 +2183,13 @@ void learn_spell_actor::info( const item &, std::vector<iteminfo> &dump ) const
                 const spell sp = pc.magic->get_spell( sp_id );
                 spell_text += ": " + string_format( _( "Level %u" ), sp.get_level() );
                 if( sp.is_max_level() ) {
-                    spell_text = "<color_light_green>" + spell_text + _( " (Max)" ) + "</color>";
+                    spell_text = string_format( "<color_light_green>%1$s (Max)</color>", spell_text );
```

Comment:
Shouldn't the string be wrapped in `_( )`? Otherwise the word "Max" will appear as-is in the UI.

```suggestion
                    spell_text = string_format( _( "<color_light_green>%1$s (Max)</color>" ), spell_text );
```